### DISCUSSION TITLE
Time Control - extend funcionality

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#17006] [General] [answered]
Personally I dont have a need to do below, but was talking with people who emphasized that this is sort of important in such a systems as Yetiforce is (as digital workforce is expanding rapidly)

SCENARIO:
Time spent on editing certain record as follow:
Module record edited (not created) > create related Time Control entry
Record saved > update related Time Control entry

Some similar CRM's have timer implemented what make time control pretty easy to adapt in above scenario.
I understand that such functionality can be customized, but maybe.... it will be good idea to set it as a core?

**Comment by rskrzypczak** [comment, marked as answer]
YetiForce has a "Working time counter" dashboard widget.
It doesn't meet the criteria you described, but there is something :)

**Reply by s2b-git** [reply]
Somehow I cannot find it - YF6.4
Any tip how to get this counter working?

**Reply by mariuszkrzaczkowski** [reply]
https://gitstable.yetiforce.com/index.php
![image](https://user-images.githubusercontent.com/10198654/229704122-43575f89-57a2-49a4-9e0b-fd67b93a8b1e.png)

**Reply by s2b-git** [reply]
Ah! Thank you. I checked and (at least here) when I start counting and navigating to some module, the timer gets stopped. Didn't managed to show counter in Demo (somehow not working)
